### PR TITLE
Add support for Alpitronic station

### PR DESCRIPTION
**Pull request by Bholtland**
- Adds a CCV Alpitronic specific configuration
- We select the configuration based on the specified ocpp version + model.
- Changed the default model value from `Chargestation v1` to `default`.
- You can now select an authorization type (RFID or CreditCard). 
- In case of Alpitronic, if the authorization type is credit card, the tagId / uid is replaced with the one provided by the CSMS.

### CODE REVIEW

File: src/lib/ChargeStation/eventHandlers/ocpp-16/ccv-alpitronic/send-data-transfer-receipt.ts [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+    receiptString: '',
+  };
+
+  chargepoint.writeCall('DataTransfer', {
+    vendorId: 'OnSitePay',
+    data: JSON.stringify(data),
+    messageId: 'Receipt',
+  });
+};
+
```
Comment (Bholtland) [inline src/lib/ChargeStation/eventHandlers/ocpp-16/ccv-alpitronic/send-data-transfer-receipt.ts:61]: Static data is returned here for now, except for the transactionId.

File: src/lib/ChargeStation/configurations/alpitronic-ccv-ocpp-16.ts [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+import sendMeterValues from '../eventHandlers/ocpp-16/send-meter-values';
+import handleReset from '../eventHandlers/ocpp-16/handle-reset';
+import handleSetChargingProfile from '../eventHandlers/ocpp-16/handle-set-charging-profile';
+import sendDataTransferAuthorize from 'lib/ChargeStation/eventHandlers/ocpp-16/ccv-alpitronic/send-data-transfer-authorize';
+import sendDataTransferReceipt from 'lib/ChargeStation/eventHandlers/ocpp-16/ccv-alpitronic/send-data-transfer-receipt';
+import handleDataTransferCallResultReceived from 'lib/ChargeStation/eventHandlers/ocpp-16/ccv-alpitronic/handle-data-transfer-call-result-received';
+
+// This is the default configuration for OCPP 1.6
+// Each key represents an event, and the value represents an array of handlers that will be called when the event is emitted
+export default {
```
Comment (lucasvmiguel) [inline src/lib/ChargeStation/configurations/alpitronic-ccv-ocpp-16.ts:35]: Could we use some sort of destruct or something to make sure once we add a handler on ocpp 1.6 we also get in here "for free"?

Comment (Bholtland) [reply src/lib/ChargeStation/configurations/alpitronic-ccv-ocpp-16.ts:35]: The problem is that this is not always desirable. Some stations don't abide to base OCPP. Alpitronic for instance does not send the `Authorize` message. We could create a bare minimum interface of some sort, but I'm not sure how helpful that's going to be.

**Review by lucasvmiguel — APPROVED** [review]

File: src/lib/ChargeStation/configurations/alpitronic-ccv-ocpp-16.ts (hunk 2 above)
Comment (lucasvmiguel) [reply src/lib/ChargeStation/configurations/alpitronic-ccv-ocpp-16.ts:35]: discussed offline. He will implement some sort of destruct:
Eg:
```js
{
...ocpp16,
[eventAlpitronic.X]: ...
[eventAlpitronic.Y]: ...
}
```

**Pull request merged**